Makefile

```diff
@@ -37,6 +37,8 @@ check_license:
 	@./scripts/check_license.sh
 
 test:
+	@echo ">> installing consul binary used in tests"
+	@$(GO) get -u github.com/hashicorp/consul
```

Comment:
This should just be vendored like the other dependencies under `vendor/`. We are currently using the `govendor` tool (https://github.com/kardianos/govendor) for that. Looking at the two new packages you are actually using, you will probably want to do:

```
go get -u github.com/hashicorp/consul/testutil github.com/hashicorp/consul/consul/structs
govendor add github.com/hashicorp/consul/testutil
govendor add github.com/hashicorp/consul/consul/structs
git add vendor/github.com/hashicorp/
git commit -m "Vendor consul packages used for testing"
```